CHANGELOG.md

```diff
@@ -2,12 +2,12 @@
 
 ### Fixes
 - Add direct requirement of `typing-extensions` ([#1586](https://github.com/neptune-ai/neptune-client/pull/1586))
+- Not coercing output of `importlib.metadata.Distributions` to `list()` ([#1612](https://github.com/neptune-ai/neptune-client/pull/1612))
```

Comment:
I think that the clue of this fix is actually to better handle distribution sorting. Not constructing a list is just a slight optimization on the side. How about this?

```suggestion
- Handle `None` values in distribution sorting in `InferDependeciesStrategy` ([#1612](https://github.com/neptune-ai/neptune-client/pull/1612))
```

Reply:
@SiddhantSadangi just this one thing and I think we can merge :)

Reply:
Or maybe we can leave it more generic as 
` - Fixed a case when dependencies were not being tracked with the "infer" strategy ([#1612](https://github.com/neptune-ai/neptune-client/pull/1612))`

wdyt?

Reply:
Works for me too, though if this strategy needs any more fixes in the future, this might prove to be a little confusing as to what exactly was fixed at any given time

Reply:
I am not sure if the issue with `None==None` existed before or was introduced by changes in this PR, hence would avoid mentioning that in the changelog

Reply:
It wouldn't exist before - if the name was `None` then sorting would error out, as in the description of the PR. But by any means you can go ahead with the more generic description

Reply:
Ah.. the `None` was leading to the sorting error!
In that case your suggestion makes sense! ✅